### PR TITLE
Migrate to standard:1.17.0 from wasm:1.15

### PR DESCRIPTION
**Pull request by nullpo-head**
This is a doc change corresponding to tetratelabs/getenvoy#114.

### CODE REVIEW

File: site/content/tutorials/getting-started-http-filter-rust.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out
    ✅ `getenvoy` will generate the actual `Envoy` config (by resolving placeholders in the configuration template )
    and start the _Envoy_ process in the foreground:
    ```
-   info	Envoy command: [$HOME/.getenvoy/builds/wasm/1.15/darwin/bin/envoy -c /tmp/getenvoy_extension_run732371719/envoy.tmpl. yaml]
+   info	Envoy command: [$HOME/.getenvoy/builds/standard/1.17.0/darwin/bin/envoy -c /tmp/getenvoy_extension_run732371719/envoy.tmpl. yaml]
```
Comment (yskopets) [inline site/content/tutorials/getting-started-http-filter-rust.md:193]: [nit] Let's remove space in `envoy.tmpl. yaml`

File: site/content/reference/getenvoy_extension_toolkit_reference.md [hunk 2]
```diff
@@ -280,7 +280,7 @@ info	Envoy command: [$HOME/.getenvoy/builds/wasm/1.15/darwin/bin/envoy -c /tmp/g
    ✅ `getenvoy` will generate the actual `Envoy` config (by resolving placeholders in the configuration template )
    and start the _Envoy_ process in the foreground:
    ```
-   info	Envoy command: [$HOME/.getenvoy/builds/wasm/1.15/darwin/bin/envoy -c /tmp/getenvoy_extension_run732371719/envoy.tmpl. yaml]
+   info	Envoy command: [$HOME/.getenvoy/builds/standard/1.17.0/darwin/bin/envoy -c /tmp/getenvoy_extension_run732371719/envoy.tmpl. yaml]
```
Comment (yskopets) [inline site/content/reference/getenvoy_extension_toolkit_reference.md:283]: [nit] Let's remove space in `envoy.tmpl. yaml`

**Review by yskopets — APPROVED** [review]
Awesome! Thank you!

File: site/content/tutorials/getting-started-http-filter-rust.md (hunk 1 above)
Comment (nullpo-head) [reply site/content/tutorials/getting-started-http-filter-rust.md]: Fixed. Thanks for the catch!

File: site/content/reference/getenvoy_extension_toolkit_reference.md (hunk 2 above)
Comment (nullpo-head) [reply site/content/reference/getenvoy_extension_toolkit_reference.md]: Fixed. Thanks for the catch!

**Comment by nullpo-head** [comment]
It looks the ci passes only after the new version of GetEnvoy CLI side is published. Will work on that including tetratelabs/getenvoy#104

**Comment by nullpo-head** [comment]
@yskopets @musaprg Since this PR regenerates the references, the changes include the documentation of `extension push`, which is being discussed in #154 by you. 
Is is OK to complete this PR including them? It seems there are still ongoing work about it.

**Comment by yskopets** [comment]
@nullpo-head I've merged #154. Hope it's enough to unblock you.

**Comment by nullpo-head** [comment]
Thanks!

**Pull request merged**